values/apl-network-policies/apl-network-policies.gotmpl

```diff
@@ -0,0 +1,6 @@
+{{- $v := .Values }}
+{{- $a := $v.apps }}
+
+# Simple per-app network policy configuration
+giteaNetworkPolicy:
```

Comment:
would you consider a similar structure to the one defined in `values/team-ns/team-ns.gotmpl`, e.g.:
```
netpols:
  gitea:
    enabled: <true|false>
```

?